pkg/agent/route/route_linux.go

```diff
@@ -878,6 +975,88 @@ func (c *Client) initServiceIPRoutes() error {
 	return nil
 }
 
+func (c *Client) initNodeNetworkPolicy() {
+	var ipProtocols []iptables.Protocol
+	if c.networkConfig.IPv4Enabled {
+		ipProtocols = append(ipProtocols, iptables.ProtocolIPv4)
+	}
+	if c.networkConfig.IPv6Enabled {
+		ipProtocols = append(ipProtocols, iptables.ProtocolIPv6)
+	}
+
+	antreaInputChainRules := []string{
+		iptables.NewRuleBuilder(antreaInputChain).
+			SetComment("Antrea: jump to static ingress NodeNetworkPolicy rules").
+			SetTarget(preNodeNetworkPolicyIngressRulesChain).
+			Done().
+			GetRule(),
+		iptables.NewRuleBuilder(antreaInputChain).
+			SetComment("Antrea: jump to ingress NodeNetworkPolicy rules").
+			SetTarget(config.NodeNetworkPolicyIngressRulesChain).
+			Done().
+			GetRule(),
+	}
+	antreaOutputChainRules := []string{
+		iptables.NewRuleBuilder(antreaOutputChain).
+			SetComment("Antrea: jump to static egress NodeNetworkPolicy rules").
+			SetTarget(preNodeNetworkPolicyEgressRulesChain).
+			Done().
+			GetRule(),
+		iptables.NewRuleBuilder(antreaOutputChain).
+			SetComment("Antrea: jump to egress NodeNetworkPolicy rules").
+			SetTarget(config.NodeNetworkPolicyEgressRulesChain).
+			Done().
+			GetRule(),
+	}
+	preIngressChainRules := []string{
+		iptables.NewRuleBuilder(preNodeNetworkPolicyIngressRulesChain).
+			MatchEstablishedOrRelated().
+			SetComment("Antrea: allow ingress established or related packets").
+			SetTarget(iptables.AcceptTarget).
+			Done().
+			GetRule(),
+		iptables.NewRuleBuilder(preNodeNetworkPolicyIngressRulesChain).
+			MatchInputInterface("lo").
+			SetComment("Antrea: allow ingress packets from loopback").
+			SetTarget(iptables.AcceptTarget).
+			Done().
+			GetRule(),
+	}
+	preEgressChainRules := []string{
+		iptables.NewRuleBuilder(preNodeNetworkPolicyEgressRulesChain).
+			MatchEstablishedOrRelated().
+			SetComment("Antrea: allow egress established or related packets").
+			SetTarget(iptables.AcceptTarget).
+			Done().
+			GetRule(),
+		iptables.NewRuleBuilder(preNodeNetworkPolicyEgressRulesChain).
+			MatchOutputInterface("lo").
+			SetComment("Antrea: allow egress packets to loopback").
+			SetTarget(iptables.AcceptTarget).
+			Done().
+			GetRule(),
+	}
+
+	for _, ipProtocol := range ipProtocols {
+		if ipProtocol == iptables.ProtocolIPv6 {
```

Comment:
```suggestion
	if c.networkConfig.IPv6Enabled {
```
and remove `ipProtocols`

Reply:
Updated.